information/services.py

```diff
@@ -0,0 +1,36 @@
+from decimal import Decimal
+import requests
+from django.core.cache import cache as cache
+from config.settings import EXCHANGE_API_URL, EXCHANGE_API_KEY, DEFAULT_CURRENCY
+
+from.validators import validate_currency_type
+
+
+def convert_to_currency(value: float, currency_to_convert: str, default_currency=DEFAULT_CURRENCY) -> Decimal:
+    """
+    Returns wallet balance converted to a given currency
+    """
+    rates = _get_conversion_rates()
+    validate_currency_type(currency_to_convert)
+    return Decimal(value*Decimal(rates[currency_to_convert])).quantize(Decimal('.01'))
+
+
+def _get_conversion_rates() -> dict:
+    """
+    Get conversion rate from exchangerate API
+    :return: dictionary with currency rates
+    """
+    if cache.get('rates'):
```

Comment:
double request to the cache. It would be better to reduce calls to the cache with usage of additional variable to store the result from the cache

Reply:
if rates := cache.get('rates'):
    return rates